Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -22,6 +22,11 @@ def linkage
       if ARGV.include?("--test")
         result.display_test_output
         Homebrew.failed = true if result.broken_dylibs?
+        if OS.linux?
+          host_whitelist = %w[libc.so.6 libm.so.6 libgcc_s.so.1 libstdc++.so.6]
```

Comment:
Should we also allow `librt.so` and `libidn.so`, since IIRC those are also part of glibc? (I forget what the version numbers for the soname are, but I think they're also 6 like the rest of them.)